crates/namada/src/ledger/mod.rs

```diff
@@ -57,8 +56,11 @@ mod dry_run_tx {
         // Wrapper dry run to allow estimating the gas cost of a transaction
         let (mut tx_result, tx_gas_meter) = match tx.header().tx_type {
             TxType::Wrapper(wrapper) => {
-                let gas_limit =
-                    Gas::try_from(wrapper.gas_limit).into_storage_result()?;
+                let gas_scale = namada_parameters::get_gas_scale(ctx.state)?;
```

Comment:
I believe we can read the parameter before the match since we need it in any case

Reply:
done in 037f62f73b6c7fd1e9a73b8d485a67caa8ec0767